osu.Game/Extensions/DrawableExtensions.cs

```diff
@@ -19,12 +22,13 @@ public static class DrawableExtensions
         /// <param name="handler">The <see cref="IKeyBindingHandler{T}"/> which is handling the repeat.</param>
         /// <param name="scheduler">The <see cref="Scheduler"/> to schedule repetitions on.</param>
         /// <param name="action">The <see cref="Action"/> to be invoked once immediately and with every repetition.</param>
+        /// <param name="initialRepeatDelay">The delay imposed on the first repeat. Defaults to <see cref="INITIAL_DELAY"/>.</param>
         /// <returns>A <see cref="ScheduledDelegate"/> which can be cancelled to stop the repeat events from firing.</returns>
-        public static ScheduledDelegate BeginKeyRepeat(this IKeyBindingHandler handler, Scheduler scheduler, Action action)
+        public static ScheduledDelegate BeginKeyRepeat(this IKeyBindingHandler handler, Scheduler scheduler, Action action, double initialRepeatDelay = INITIAL_DELAY)
```

Comment:
I suppose... But we can't be having this in the future, when `KeyBindingContainer` has repeat support. These values match the ones in `InputManager` and in general I disagree with having different repeat delays for different components.

Reply:
The one in `InputManager` feels way too slow, so either we decide to change it game-wide (not sure I agree with this... we should probably be reading from the OS where feasible) or having overrides like this..

Reply:
My OS repeat delay is double this value, and much closer to the 250 value, so this feels way too fast for me. Definitely don't agree with this, but agree with using OS values if possible and a game-wide setting if not.